--- a/jhbuild/environment.py	2025-07-18 02:10:36
+++ b/jhbuild/environment.py	2025-07-18 02:15:47
@@ -21,7 +21,11 @@
 
 import sys
 import os
-from distutils.sysconfig import get_python_lib
+
+if sys.version_info.major > 3 or sys.version_info.minor >= 12:
+    from setuptools._distutils import sysconfig
+else:
+    from distutils import sysconfig
 
 from jhbuild.utils.cmds import get_output
 
@@ -193,7 +197,7 @@
     # to lib, so any distutils based build will install .pc files
     # into lib64. To make at least pkg-config happy add the host
     # libdir layout as well.
-    host_libdir = os.path.dirname(get_python_lib(True, True, prefix))
+    host_libdir = os.path.dirname(sysconfig.get_python_lib(True, True, prefix))
     host_pkgconfigdir = os.path.join(host_libdir, 'pkgconfig')
     addpath('PKG_CONFIG_PATH', host_pkgconfigdir)
 
