--- a/CMakeLists.txt	2025-11-09 17:22:13
+++ b/CMakeLists.txt	2025-11-09 17:21:04
@@ -155,6 +155,7 @@
 target_include_directories(zlibstatic PUBLIC ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 set_target_properties(zlib PROPERTIES DEFINE_SYMBOL ZLIB_DLL)
 set_target_properties(zlib PROPERTIES SOVERSION 1)
+set_target_properties(zlib PROPERTIES INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
 
 if(NOT CYGWIN)
     # This property causes shared libraries on Linux to have the full version
