--- CMakeLists.txt.ori	2026-05-10 14:31:12
+++ CMakeLists.txt	2026-05-10 14:32:21
@@ -184,6 +184,11 @@
         PROPERTIES DEFINE_SYMBOL ZLIB_DLL
                    EXPORT_NAME ZLIB
                    OUTPUT_NAME z)
+
+    set_target_properties(
+        zlib
+        PROPERTIES INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+
     if(UNIX
         AND NOT APPLE
         AND NOT (CMAKE_SYSTEM_NAME STREQUAL AIX)
